diff --git a/vendor/smalot/pdfparser/src/Smalot/PdfParser/Page.php/Page.php b/vendor/smalot/pdfparser/src/Smalot/PdfParser/Page.php
index 1bd29e1..9a347be 100644
--- a/vendor/smalot/pdfparser/src/Smalot/PdfParser/Page.php
+++ b/vendor/smalot/pdfparser/src/Smalot/PdfParser/Page.php
@@ -287,7 +287,9 @@ class Page extends PDFObject
     {
         $pageNum = $this->getPageNumber();
         $xObjects = $this->getXObjects();
-
+        if ($pageNum === 0) {
+            $pageNum = 'TPL1';
+        }
         return $xObjects[$pageNum];
     }
 
